### PR TITLE
[BP-3] Add --identity option to CLI syntax

### PR DESCRIPTION
**Pull request by sgregglives**
Signed-off-by: Steven Gregg <steven.gregg@consensys.net>

## PR description
Add description of `--identity` CLI option
## Fixed Issue(s)

BP-3

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -276,6 +276,26 @@ By default, access from `localhost` and `127.0.0.1` is accepted.
...
+
+```bash tab="Environment Variable"
+BESU_IDENTITY=MyNode
+```
+
+```bash tab="Configuration File"
+identity="MyNode"
+```
+
+A name for the node. If specified, it is the second part of the client ID provided by some Ethereum network explorers. For example, in the client ID `besu/MyNode/v1.3.4/linux-x86_64/oracle_openjdk-java-11`, the node name is `MyNode`.
```
Comment (MadelineMurray) [inline docs/Reference/CLI/CLI-Syntax.md]: A name for the node -> Name for the node

Nitpick: A is redundant

Comment (sgregglives) [reply docs/Reference/CLI/CLI-Syntax.md]: Updated.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -276,6 +276,26 @@ By default, access from `localhost` and `127.0.0.1` is accepted.
...
+```bash tab="Environment Variable"
+BESU_IDENTITY=MyNode
+```
+
+```bash tab="Configuration File"
+identity="MyNode"
+```
+
+Name for the node. If specified, it is the second part of the client ID provided by some Ethereum network explorers. For example, in the client ID `besu/MyNode/v1.3.4/linux-x86_64/oracle_openjdk-java-11`, the node name is `MyNode`.
+
```
Comment (shemnon) [inline docs/Reference/CLI/CLI-Syntax.md]: I think we need some verbiage to set expectations if it is not set.
```suggestion
If this value is not set then the client ID will not have a node name section, for example `besu/v1.3.4/linux-x86_64/oracle_openjdk-java-11`.
```

Comment (sgregglives) [reply docs/Reference/CLI/CLI-Syntax.md]: I revised your suggestion to conform with some of our doc standards.

**Pull request merged**